app/controllers/invitations/psp/CheckYourAnswersController.scala

```diff
@@ -60,12 +59,10 @@ class CheckYourAnswersController @Inject()(override val messagesApi: MessagesApi
     implicit request =>
       (PspNameId and PspId).retrieve.map {
         case pspName ~ pspId =>
-          minimalConnector.getNameFromPspID(pspId).map {
-            case Some(minPspName) if pspAuthoriseFuzzyMatcher.matches(pspName, minPspName) =>
-              Redirect(routes.DeclarationController.onPageLoad(srn))
-            case _ => Redirect(routes.PspDoesNotMatchController.onPageLoad(srn))
+          minimalConnector.getEmailInvitation(pspId, "pspid", pspName, srn).map { email =>
+                Redirect(routes.DeclarationController.onPageLoad(srn))
           }.recoverWith {
-            case _: DelimitedPractitionerException => Future.successful(Redirect(routes.PspDoesNotMatchController.onPageLoad(srn)))
```

Comment:
Probably want to keep this DelimitedPractitionerException

Reply:
I can add it back but this Exception was throwing by an old endpoint here: https://github.com/hmrc/manage-pensions-frontend/blob/43fa4941f33029746f0aa86a2f032053da0fb5a6/app/connectors/admin/MinimalConnector.scala#L88  but now we replaced this with email invite endpoint which will not throw this exception (https://github.com/hmrc/manage-pensions-frontend/pull/635/files#diff-867241724ef1fb274c0d38ec65c3bc89dd382446d8c9576e8d8582dadb61d8baR63 ).

Reply:
It will still throw that exception.
That one comes from IF server.

Reply:
Ok, I will add that. Thanks